src/assets/config_files/usecases.json

```diff
@@ -323,6 +323,9 @@
         "experience": ["all"],
         "maturity": ["beta"],
         "disabled": false,
+        "contributors": [
+          {"name": "David C Sterratt", "email": "david.c.sterratt@ed.ac.uk"},
```

Comment:
Please see that the validation failed due there is a training comma (JSON does not allow that).
Could you please try to amend this?

Reply:
I tried to push the changes, but I ran in the usual git error

error: failed to push some refs to 'git@github.com:alex4200/usecases.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

any idea how to do it right?